src/Admin/Settings.php

```diff
@@ -506,27 +500,26 @@ public function wpgraphql_admin_table_columns_html( string $column_name, int $po
 
 		if ( empty( $post_id ) ) {
 			echo null;
+			return;
 		}
 
-		// @phpstan-ignore-next-line
 		$field_group = acf_get_field_group( $post_id );
 
 		if ( empty( $field_group ) ) {
 			echo null;
+			return;
 		}
 
 		switch ( $column_name ) {
 			case 'acf-wpgraphql-type':
 				$type_name = $this->get_registry()->get_field_group_graphql_type_name( $field_group );
 
-				// @phpstan-ignore-next-line
 				echo '<span class="acf-wpgraphql-type">' . acf_esc_html( $type_name ) . '</span>';
```

Comment:
What's the intended behavior here - for the `<span>` to be empty or for the span not to get printed altogether?